server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/BUILD

```diff
@@ -6,16 +6,11 @@ java_library(
     visibility = ["//server:__subpackages__"],
     deps = [
         "//bazelrunner",
-        "//commons",
-        "//executioncontext/workspacecontext",
-        "//server/src/main/java/org/jetbrains/bsp/bazel/server/bep",
-        "//server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/managers",
         "//server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/services",
-        "//server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/utils",
         "//server/src/main/java/org/jetbrains/bsp/bazel/server/sync",
-        "@io_bazel//src/main/protobuf:build_java_proto",
         "@io_bazel//third_party/grpc:grpc-jar",
         "@maven//:ch_epfl_scala_bsp4j",
+        "@maven//:io_vavr_vavr",
```

Comment:
it didn't depend on vavr and now it does?

Reply:
mine fuckup with exports xd